web/common.h

```diff
@@ -0,0 +1 @@
+#define MSG_NOSIGNAL 0 
```

Comment:
That should only be there for systems that have MSG_NOSIGNAL, Basically anything non-windows.  I am pretty sure OSX has that!